flake.nix

```
@@ -1,75 +1,21 @@
 {
-    inputs = {
-        # Here you can adjust the IHP version of your project
-        # You can find new releases at https://github.com/digitallyinduced/ihp/releases
-        ihp.url = "github:digitallyinduced/ihp?ref=80373b24fca3bdb03c0f5da6075cc84520938f8f";
-        ihp.flake = false;
+    inputs.ihp.url = "github:digitallyinduced/ihp/nicolas/flake";  # TODO branch/release
 
-        # See https://ihp.digitallyinduced.com/Guide/package-management.html#nixpkgs-pinning
-        nixpkgs.url = "github:NixOS/nixpkgs?rev=a95ed9fe764c3ba2bf2d2fa223012c379cd6b32e";
+    outputs = { self, ihp }: ihp.inputs.flake-parts.lib.mkFlake { inherit (ihp) inputs; } {
```

```suggestion
    outputs = inputs@{ self, ihp }: ihp.inputs.flake-parts.lib.mkFlake { inherit inputs; } {
```

That was a very dirty hack that seems to be even more dirty than I thought, and I don't think I need it at all. Look at that again.